common/lib/dependabot/clients/azure.rb

```diff
@@ -22,7 +22,7 @@ class TagsCreationForbidden < StandardError; end
 
       RETRYABLE_ERRORS = [InternalServerError, BadGateway, ServiceNotAvailable].freeze
 
-      MAX_PR_DESCRIPTION_LENGTH = 3999
+      PR_DESCRIPTION_MAX_LENGTH = 3999
```

Comment:
```suggestion
```

Reply:
Yes that was a miss.  It should no longer be in the Client.